java/code/src/com/suse/manager/webui/controllers/AdminConfigController.java

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
This folder webui/controllers is growing in an uncontrolled way, it's hard to understand which code it's related or not. 

What about creating a new folder for this feature, having a structure like this: https://github.com/uyuni-project/uyuni/tree/master/java/code/src/com/suse/manager/webui/controllers/contentmanagement

You could even follow the same folder structure as you already have in the frontend "admin/config" and move the controller and new templates there.